RecoParticleFlow/PFProducer/interface/PFEGammaFilters.h

```diff
@@ -57,6 +57,10 @@ class PFEGammaFilters {
   float ele_dnnLowPtThr_;
   float ele_dnnHighPtBarrelThr_;
   float ele_dnnHighPtEndcapThr_;
```

Comment:
All these `*Thr_` , as well as the `ele_max*` and most of the `badHcal_*` ones, can be made `const`

Reply:
We kept them non-const only because they are read from a group of options in the constructor https://github.com/cms-sw/cmssw/blob/master/RecoParticleFlow/PFProducer/src/PFEGammaFilters.cc#L72 and I felt it would be ugly to put that in the initializer list. How do you suggest to fix this problem? Thanks

Reply:
Uhm, yes, it would require some gymnastic indeed...
Ok, don't mind